### PR TITLE
Feature/grants page

### PR DESCRIPTION
**Pull request by bwkarr77**
# Description

Fixed some minor bugs and styling.
Added in sections for logic testing.
Starting working on implementing Global state into Homepage.

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Change Status

- [x] Complete, but not tested (may need new tests)

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [x] My changes generate no new warnings
- [x] There are no merge conflicts

### CODE REVIEW

File: src/components/homepage/Homepage.jsx [hunk 1]
```diff
@@ -27,9 +27,23 @@ const Homepage = () => {
   const grants = useSelector((state) => {
     return state.homePage.grantsInfo;
   });
+  const state = useSelector((state) => {
+    return state;
+  });
   const user = useSelector((state) => {
     return state.homePage.userInfo;
   });
+  const testUser = useSelector((state) => {
```
Comment (FreedomWriter) [inline src/components/homepage/Homepage.jsx:36]: I can't seem to figure out where this testUser is being used, could you help clear it up for me?

File: src/components/homepage/Homepage.jsx [hunk 2]
```diff
@@ -41,7 +55,9 @@ const Homepage = () => {
   //=====================
 
   console.log("Homepage: user, grants: ", user, grants);
-  console.log("Homepage: status: ", status);
+  console.log("Homepage: state: ", state.login.usertype);
```
Comment (FreedomWriter) [inline src/components/homepage/Homepage.jsx:58]: let's make sure to get rid of the console.logs

File: src/store/actions/HomepageActions.js [hunk 3]
```diff
@@ -67,24 +67,18 @@ export const getUserInfo = (info) => (dispatch) => {
   axios
     .get(`${userInfo}`)
     .then((res) => {
```
Comment (FreedomWriter) [inline src/store/actions/HomepageActions.js:69]: love a good bit of cleanup

File: src/components/homepage/Homepage.jsx (hunk 1 above)
Comment (bwkarr77) [reply src/components/homepage/Homepage.jsx:36]: Short answer: it's not doing anything.....yet O.o 
Currently user is being set with hard coded data, but testuser is trying to set the user based on the state that's returned when the user logs in.

**Pull request merged**